db/migrate/20220121105031_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.integer :authorid
+      t.string :title
+      t.text :text
+      t.integer :commentscounter
+      t.integer :likescounter
```

Comment:
- [ ] It will be nice to follow ruby's best practice for naming convention. The reason is given above.

Reply:
This issue is well resolved! ✔️